src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -318,7 +313,7 @@ def cli_cosmosdb_network_rule_remove(cmd,
         else:
             rule_removed = True
     if not rule_removed:
-        raise CLIError("This rule does not exist for the Cosmos DB account")
+        raise logger.warning("This rule does not exist for the Cosmos DB account")
```

Comment:
This actually would be okay to remain an error, but the end result isn't any different. Deletes on non-existent resources throw errors.

Reply:
I'll make the change so it is consistent